packages/metadata/src/metadata-service.ts

```diff
@@ -45,10 +45,10 @@ function getActionName(name: Path): Path {
 
 /**
  *
- * @param metadata
- * @param actionNames
- * @param namespaces
- * @param metadataElements
+ * @param metadata metadata element map
```

Comment:
We should probably use the same convention as in other packages where descriptions start with `-` character.